packages/react-native-media-player/ios/MediaPlayer.swift

```diff
@@ -0,0 +1,30 @@
+import AVFoundation
+
+@objc(MediaPlayer)
+public class MediaPlayer: NSObject {
+
+    var avPlayer:AVAudioPlayer!
+  @objc(load:resolver:rejecter:)
+  public func load(to url: String, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) {
+    // Date is ready to use!
+    do {
+        let fileURL = URL(string:url);
+        let soundData = try Data(contentsOf:fileURL!)
+        avPlayer = try AVAudioPlayer(data: soundData)
+        resolve(NSNull());
+    } catch {
+        reject("Failed", "Something went wrong", error);
+    }
+  }
+  
+  @objc(pause)
+  public func pause() {
+    avPlayer.pause();
+  }
+
+  @objc(play)
+  public func play() {
+    avPlayer.play();
```

Comment:
Codacy found an issue: [Lines should not have trailing semicolons.](https://app.codacy.com/manual/yajananrao/Nadha/pullRequest?prid=5728824)